frontend/src/components/interview/interview.tsx

```diff
@@ -75,6 +79,16 @@ const Interview: FC = () => {
         }),
       );
     }
+
+    if (hasManageInterviewPermissions && !hasPermissionToSelectInterviewer) {
+      dispatch(
+        interviewActions.checkUserIsInterviewerOrInterviewee({
+          interviewee: interview?.interviewee.id as number,
+          interviewer: interview?.interviewer?.id as number,
+          userId: user?.id as number,
+        }),
+      );
```

Comment:
Can't you just check if this user is related to current interview inside this component instead of having such amount of extra steps 🤔

Reply:
agreed